cilium-cli/clustermesh/clustermesh.go

```diff
@@ -80,7 +80,7 @@ type k8sClusterMeshImplementation interface {
 	CreateCiliumExternalWorkload(ctx context.Context, cew *ciliumv2.CiliumExternalWorkload, opts metav1.CreateOptions) (*ciliumv2.CiliumExternalWorkload, error)
 	DeleteCiliumExternalWorkload(ctx context.Context, name string, opts metav1.DeleteOptions) error
 	ListCiliumEndpoints(ctx context.Context, namespace string, options metav1.ListOptions) (*ciliumv2.CiliumEndpointList, error)
-	CiliumLogs(ctx context.Context, namespace, pod string, since time.Time, previous bool) (string, error)
+	CiliumLogs(ctx context.Context, namespace, pod, containerName string, since time.Time, previous bool) (string, error)
```

Comment:
nit: should we rename it to "containerLogs" now though?

Reply:
Sure. I stacked a new commit on top for this.